File: c#2010/PDFFormFields/FormTextBox.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Windows.Forms;

namespace WindowsApplication1
{
    public partial class FormTextBox : Form
    {
        public FormTextBox()
        {
            InitializeComponent();
        }

        public int FieldIndex
        {
            set { lblindex.Text = value.ToString(); }
        }

        public string Value
        {
            get { return textBox1.Text; }
            set { textBox1.Text = value; }
        }

        private void button1_Click(object sender, EventArgs e)
        {
            this.DialogResult = DialogResult.OK;
            this.Close();
        }
    }
}

File: VBdotNet2010/PDFFormFields/FormTextBox.vb
Public Class FormTextBox

    Public Property FieldIndex() As Integer
        Get
            Return Val(lblindex.Text)

        End Get

        Set(ByVal value As Integer)

            lblindex.Text = value
        End Set

    End Property


    Public Property Value() As String
        Get
            Return textBox1.Text

        End Get

        Set(ByVal myValue As String)

            textBox1.Text = myvalue
        End Set

    End Property

    Private Sub FormTextBox_Load(sender As System.Object, e As System.EventArgs) Handles MyBase.Load

    End Sub

    Private Sub button1_Click(sender As System.Object, e As System.EventArgs) Handles button1.Click
        DialogResult = DialogResult.OK
        Close()
    End Sub
End Class